docs/topics/db/queries.txt

```diff
@@ -985,6 +985,15 @@ entry::
     entry.save()
     entry.authors.set(old_authors)
 
+For a ``OneToOneField``, you must duplicate the related object to avoid
+violating the one-to-one unique constraint. For example, assuming ``entry`` is
+already duplicated as above::
+
+    detail = EntryDetail.objects.all()[0]
+    detail.pk = None
+    detail.entry = entry
```

Comment:
I had that in the first draft but it seemed obvious so I removed it. I'll add it back.